nva-datamodel-java/src/main/resources/publication-ontology.ttl

```diff
@@ -0,0 +1,1744 @@
+@prefix nva: <https://nva.sikt.no/ontology/publication#> .
```

Comment:
Did you write all this manually?

Reply:
Nope, I generated the majority of it — but had to do a lot of small fixes manually

Reply:
In the future, I suspect we will have some annotations that aid generation.